HW-0.Rmd

```diff
@@ -9,6 +9,7 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](http://rpubs.com/cmernestein/hw-0)
 * Click Knit HTML
 * Please indicate how much time you spent on this assignment: 
+1.5 hr
```

Comment:
You included the wrong rpubs link, but that's fine. The point of this HW-0 is the submission, not the content of the html file.

Also 1.5 hour is a little longer than I envisioned for students. If you get stuck with Git issues later, please come see me during office hours.